pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/fullsecretmanagersecret-auto-legacy/_generated_object_fullsecretmanagersecret-auto-legacy.golden.yaml

```diff
@@ -1,40 +0,0 @@
-apiVersion: secretmanager.cnrm.cloud.google.com/v1beta1
```

Comment:
Normally I would say we want to convert this test to use direct, but I imagine the argument is that then it duplicates the `-direct` test.

I propose we accept that the tests are duplicates.  Then we have a separate automated process that comes in and looks for duplicate tests and proposes deletion.  So when we have gotten everything to direct if we see a `-legacy` test (or a `-direct` test) we know something has gone wrong with the merging and handle it then.